hetzner_invoice/etl.py

```diff
@@ -18,6 +18,5 @@
     user=os.getenv("HETZNER_INVOICE_DB_USER"),
     pw=os.getenv("HETZNER_INVOICE_DB_PW"),
     host=os.getenv("HETZNER_INVOICE_DB_HOST"),
-    port=os.getenv("HETZNER_INVOICE_DB_PORT"),
```

Comment:
It would be nice to keep the port configuration because not everyone is using the default port.